app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -99,7 +99,45 @@ export default {
   },
   computed: {
     message() {
-      return this.formatMessage(this.data.content, this.isATweet);
+      let messageContent = this.formatMessage(this.data.content, this.isATweet);
+      if (this.contentType === 'input_select') {
+        const {
+          submitted_values: submittedValues = [],
+        } = this.contentAttributes;
+        const [selectedOption] = submittedValues;
+
+        if (selectedOption && selectedOption.title) {
+          messageContent += `<strong>${selectedOption.title}</strong>`;
```

Comment:
If this parsing is before, can we use markdown instead of HTML ?

Reply:
The content becomes unreadable with all the \n and spaces. I would prefer to go with this.